src/crimson/os/seastore/omap_manager/btree/string_kv_node_layout.h

```diff
@@ -786,14 +786,15 @@ class StringKVInnerNodeLayout {
   static std::string balance_into_new_nodes(
     const StringKVInnerNodeLayout &left,
     const StringKVInnerNodeLayout &right,
+    uint32_t pivot_idx,
     StringKVInnerNodeLayout &replacement_left,
     StringKVInnerNodeLayout &replacement_right)
   {
+    ceph_assert(!left.below_min() || !right.below_min());
```

Comment:
IIUC this can be parsed as: if both left and right are below min, they should be merged rather than rebalanced, right?

So,
```
ceph_assert(!(left.below_min() && right.below_min()));
```
can look more straightforward without documentation.

Reply:
Was `ceph_assert(!(left.below_min() && !right.below_min()));`